help-jellyfin/index.html.bak

```diff
@@ -0,0 +1,439 @@
+<!DOCTYPE html>
```

Comment:
why add this file?

Reply:
Looks like @joshuaboniface used sed at some point and forgot to remove this file.

Reply:
Aah yes I did, thanks.